output/opendap_post2.sh

```diff
@@ -77,9 +89,29 @@ if [[ ${#SERVERS[@]} -eq 0 ]]; then
    fi
    exit
 fi
+
+#+ vvvvv
+# BEGIN - determine what files to upload, takes into
+# consideration files written in the scenario's run.properties
+# file, using 2 keys:
+#   + post.opendap.files           - generated by output/createOPeNDAPFileList.sh
+#   + post.opendap.additionalFiles - specified by user config via "postAdditionalFiles"
+#
+declare -a FILES_Standard
+fileList=${properties["post.opendap.files"]}
```

Comment:
For reasons described above, I suggest
```
declare -a fileList
fileList=( $( echo ${properties["post.opendap.files"]} | tr -d "()") )
```
Not sure the `declare` is even needed.

Reply:
FWIW, you (assuming) have accounted for this where `$FILES` are looped over,

```
      while [[ $fileIndex -lt `expr ${#FILES[@]} - 1` ]] ; do  # skip the closing "("
         file="${FILES[$fileIndex]}"
         if [[ $file = '(' || $file = ')' || ! -s $file ]]; then
            fileIndex=`expr $fileIndex + 1` 2>> $SCENARIOLOG
            continue
         fi
 ...
```